examples/globe_wfs_extruded.html

```diff
@@ -94,7 +94,7 @@
                 name: 'lyon_tcl_bus',
                 type: 'geometry',
                 update: itowns.FeatureProcessing.update,
-                convert: itowns.Feature2Mesh.convert({
+                convert: itowns.Feature2Mesh({
```

Comment:
Little detail, but can we then change the name of the export to `feature2Mesh` without capitalization?

Reply:
Yep